web/beacon-app/src/features/teams/components/ChangeRoleModal.tsx

```diff
@@ -41,6 +41,10 @@ function ChangeRoleModal({ openChangeRoleModal, setOpenChangeRoleModal }: Change
             setOpenChangeRoleModal({ ...openChangeRoleModal, opened: false });
           }
         },
+        onSuccess() {
+          toast.success('Success! You have updated your teammate’s role in your organization.');
```

Comment:
```suggestion
          toast.success('The team member’s role is updated.');
```

Reply:
I saw this success message in the Figma file.

Reply:
This success message is from Edwin in our last call